include/aikido/util/CatkinResourceRetriever.hpp

```diff
@@ -1,9 +1,9 @@
 #ifndef AIKIDO_UTIL_CATKINRESOURCERETRIEVER_HPP_
 #define AIKIDO_UTIL_CATKINRESOURCERETRIEVER_HPP_
 #include <string>
-#include <unordered_map>
 #include <vector>
 #include <dart/common/ResourceRetriever.hpp>
+#include <unordered_map>
```

Comment:
I also prefer that way. ClangFormat supports this feature, and I thought this is addressed by [this]. Let me double check.